packages/core/src/sequence.ts

```diff
@@ -35,11 +72,21 @@ export class Sequence {
       this.sendError(res, req, err);
     }
   }
-
+   /**
+   * Sends API result in HTTP response
```

Comment:
This is only partially true. What if I produce the value in the sequence itself? We need to err on the side o flonger descriptions. Short descriptions like this are pretty confusing.